scripts/jenkins/jobs-ibs/cloud-mediacheck.yaml

```diff
@@ -63,6 +64,11 @@
               export RPM_DEPCHECK_MEDIADEPS=" "
               export SRCRPM_DEPCHECK_MEDIADEPS=" "
               export INSTALLCHECK_EXTRA_OPTS="--exclude pattern:"
+          elif [[ "$project" =~ Devel:Cloud:(.*):Appliances ]]; then
+              cloud_version=${BASH_REMATCH[1]}
+              export RPM_DEPCHECK_MEDIADEPS="solv/cloud-$cloud_version-dvd.solv solv/cloud-$cloud_version-updates.solv"
```

Comment:
> is it actually possible to combine SES with Cloud? I thought there were a few python packages in different versions needed.

There's one conflict (django), which means we have to sacrifice calamari. But that's reasonable for the appliance, I think.

> main downside is that we're loosing the extra skelcds and the license information I guess...

Hrm. Might be an issue, es...